lib/core/elastic_connector_actions.rb

```diff
@@ -157,28 +157,25 @@ def claim_job(connector_id)
           seq_no,
           primary_term
         )
+      end
 
+      def create_job(connector_settings:)
         body = {
-          :status => Connectors::SyncStatus::IN_PROGRESS,
-          :worker_hostname => Socket.gethostname,
-          :created_at => Time.now,
-          :started_at => Time.now,
-          :last_seen => Time.now,
-          :connector => {
-            :id => connector_id,
-            :filtering => convert_connector_filtering_to_job_filtering(connector_record.dig('_source', 'filtering'))
+          status: Connectors::SyncStatus::PENDING,
+          worker_hostname: Socket.gethostname,
+          created_at: Time.now,
+          started_at: Time.now,
+          last_seen: Time.now,
+          connector: {
+            id: connector_id,
+            filtering: convert_connector_filtering_to_job_filtering(connector_settings.filtering)
```

Comment:
We should also add `configuration`, `pipeline`, `index_name`, `language` and `service_type`

Reply:
yep, this PR is draft so don't consider is complete